netlify/transDimConfig.js

```diff
@@ -0,0 +1,34 @@
+const transDimConfig = {
+   // SMTP SETTINGS of server that will handle the actual sending of the emails
+   smtp: {
+      host: process.env.SMTP_HOST,
```

Comment:
Could we provide defaults for some of these (obvs not the secret ones).

Might be irrelevant to do so, but I usually want to have fallbacks for anything that is in a config.

Reply:
see earlier comment